e2e-tests/cypress/integration/guidance-page-before-you-appeal.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Guidance Page - Before you Appeal
+  As a PO on the appeal service
+  I want prospective appellants to be aware of who the service is for
+  So that only those who need to appeal through the new appeal service do so
+
+  Scenario: Appellant navigates to next page
+    Given the appellant is on the before you appeal page
+    When the appellant navigates to the next page
+    Then information about when you can appeal is provided
+
+  Scenario Outline: Appellant selects <page> page from the Content list
+    Given the appellant is on the before you appeal page
+    When the appellant select a link from the content list: <page>
+    Then the appellant is navigated to that page: <url>
+
```

Comment:
Please check with @pippalou , that the current page shouldn't be a link in content list and we should check for it.

Reply:
This is the behaviour in the prototype

Reply:
I agree, I was in a joint review with @pippalou and thus I have asked you to check with her.